DOCS/interface-changes/sub-ass-use-video-data.txt

```diff
@@ -0,0 +1,16 @@
+Remove sub-ass-vsfilter-aspect-compat.
+Remove sub-ass-vsfilter-blur-compat.
+Add sub-ass-use-video-data.
+Change default V keybind to cycle sub-ass-use-video-data
+instead of toggling the now removed sub-ass-vsfilter-aspect-compat.
+
+The naming for "blur-compat" was misleading since the setting
+actually affects more than just blur affects. Additionally
+forwarding storage resolution but forcing an aspect ratio
+of 1.0 for the video is likely to result in odd rendering
+and there’s no known usecase for it.
+
+The new option is a tri-state of passing either no video information,
+only aspect ratio, or both aspect ratio and storage resolution.
+By default all available info is passed matching the previous defaults
+and what’s required for standard ASS rendering.
```

Comment:
This will eventually be merged into https://github.com/mpv-player/mpv/blob/master/DOCS/interface-changes.rst file. I appreciate the detailed explanation, but I think it should be more concise to not overshadow other changes.

I would say first 5 lines is enough, the rest can be inferred from new docs.

Reply:
done